src/test/java/unit/uk/gov/ida/verifyserviceprovider/services/ClassifyingAssertionTranslatorTest.java

```diff
@@ -32,83 +28,25 @@
     @Mock
     private VerifyAssertionTranslator verifyAssertionService;
 
-    @Mock
-    private EidasAssertionTranslator eidasAssertionService;
-
-    @Mock
-    private EidasUnsignedAssertionTranslator eidasUnsignedAssertionTranslator;
-
     @Before
     public void setUp() {
         initMocks(this);
```

Comment:
Is `initMocks()` required? If not the whole `setUp` method could go.

Reply:
The tests fail if initMocks isn't there